orchagent/fdborch.h

```diff
@@ -113,6 +113,7 @@ class FdbOrch: public Orch, public Subject, public Observer
     Table m_mclagFdbStateTable;
     NotificationConsumer* m_flushNotificationsConsumer;
     NotificationConsumer* m_fdbNotificationConsumer;
+    shared_ptr<DBConnector> m_notificationsDb;
```

Comment:
Is it really necessary to introduce new member?

Reply:
Yes. The lifetime of `m_notificationsDb` needs to be at least as long as `m_fdbNotificationConsumer`, since `m_fdbNotificationConsumer` uses `m_notificationsDb`. Because currently it's just a local variable with a pointer, and because `swss::NotificationConsumer` doesn't accept `shared_ptr`, we need to make sure that `shared_ptr` doesn't free `DBConnector` too early.